src/common/actions/general.actions.ts

```diff
@@ -42,7 +42,7 @@ import {
 } from '../types/actions.types';
 import { ParachainStatus, Prices } from '../types/util.types';
 
-export const isPolkaBtcLoaded = (isLoaded = false): IsPolkaBtcLoaded => ({
+export const isBridgeLoaded = (isLoaded = false): IsPolkaBtcLoaded => ({
   type: IS_POLKA_BTC_LOADED,
   isLoaded
```

Comment:
Please rename also the `IsPolkaBtcLoaded` type and action  `IS_POLKA_BTC_LOADED`.

Reply:
Thanks. Done!